src/index.js

```diff
@@ -19,6 +19,15 @@ import DataExplorer from './containers/DataExplorer';
 
 const store = configureStore();
 
+import messages from './messages'; // Lang does not know where did you get your messages from.
+
+import LangSugar from './lang';
+window.L = new LangSugar();
```

Comment:
I considered context too, as many other libraries use (as I stated on #9), I think it introduces complexity and is highly discouraged by the React team: https://facebook.github.io/react/docs/context.html

I know globals are not pretty, but I think a global feature like i18n is one of the few cases where I would accept it. Other ways lead to more boilerplate on every component.

This could change in the future, I may add more logic to the decorator to strengthen the implementation.